dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -245,6 +245,11 @@ DSRA4004.onconnect.sql=DSRA4004E: Unable to execute onConnect SQL command \"{0}\
 DSRA4004.onconnect.sql.explanation=An error occurred while attempting to execute the specified SQL command.
 DSRA4004.onconnect.sql.useraction=Refer to the exception message to help determine the cause of the problem.
 
+#4008			TRANSACTION_NONE_UNSUPPORTED
+TRANSACTION_NONE_UNSUPPORTED=DSRA4008: The JDBC driver does not support an isolation level of TRANSACTION_NONE.  The isolation level was set to TRANSACTION_NONE because of the data source configuration.
```

Comment:
Update the message key to match that of the preceding keys (DSRA4008.description) as that makes it easier to search for the keys in files.

Reply:
Also the explanation and message shouldn't be quite so similar (and could use a little more detail).  Maybe something more along the lines of:
message: DSRA4008: The datasource {0} was configured with an isolation level of TRANSACTION_NONE, but that isolation level is not supported by the JDBC driver.
explanation: The datasource was configured with with an isolation level of TRANSACTION_NONE, but that is not the JDBC driver's default isolation level and the Application Server was unable to set the driver's isolation level to TRANSACTION_NONE.